checks.d/mongo.py

```diff
@@ -619,6 +636,7 @@ def check(self, instance):
                     data['health'] = current['health']
 
                 data['state'] = replSet['myState']
+                tags.append('mongo_replset_state:%s' % self.REPLSET_STATES[data['state']])
```

Comment:
Given that this is the only check using this tag, it's unnecessary to specify this as `mongo_*`. Could we change this instead to `replset_state`?